store/sqlstore/post_store.go

```diff
@@ -991,13 +991,17 @@ func (s *SqlPostStore) AnalyticsUserCountsWithPostsByDay(teamId string) (model.A
 	return rows, nil
 }
 
-func (s *SqlPostStore) AnalyticsPostCountsByDay(teamId string) (model.AnalyticsRows, *model.AppError) {
+func (s *SqlPostStore) AnalyticsPostCountsByDay(teamId string, botsOnly bool, yesterdayOnly bool) (model.AnalyticsRows, *model.AppError) {
```

Comment:
Could we introduce an `AnalyticsPostCountsOptions` struct to capture these bits? I find that clearer to follow than random `true, false` parameters without immediate context into what they mean.

Reply:
Absolutely.  I'll add to the next commit and keep this in mind with creating functions that have multiple inputs.

Reply:
I'm wondering if `AnalyticsPostCountsByDay` should /accept/ the `AnalyticsPostCountsOptions` -- otherwise, the caller is still left passing `true, false` as such, where the confusion is likely to arise. Maybe `AnalyticsPostCountsOptions` could go into the `model` package alongside our other `*Options`-style structs?